src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -2593,6 +2607,208 @@ public override IOperation VisitLock(ILockOperation operation, int? captureIdFor
             return null;
         }
 
+        public override IOperation VisitForEachLoop(IForEachLoopOperation operation, int? captureIdForResult)
+        {
+            Debug.Assert(_currentStatement == operation);
+
+            ForEachLoopOperationInfo info = ((BaseForEachLoopStatement)operation).Info;
+
+            bool haveLocals = !operation.Locals.IsEmpty;
+            bool createdRegionForCollection = false;
+
+            if (haveLocals && operation.LoopControlVariable.Kind == OperationKind.VariableDeclarator)
+            {
+                // VB has rather interesting scoping rules for control variable.
+                // It is in scope in collection expression. However, it is considered to be 
```

Comment:
>  [](start = 36, length = 1)

Typo: missing `the`

Reply:
And again 2 lines down

---
In reply to: [181888492](https://github.com/dotnet/roslyn/pull/26158#discussion_r181888492) [](ancestors = 181888492)